vassal-app/src/main/java/VASSAL/build/module/Map.java

```diff
@@ -225,13 +225,13 @@
   protected String markUnmovedText = ""; //$NON-NLS-1$
   protected String markUnmovedTooltip = Resources.getString("Map.mark_unmoved"); //$NON-NLS-1$
   protected MouseListener multicaster = null;
-  protected ArrayList<MouseListener> mouseListenerStack = new ArrayList<>();
+  protected List<MouseListener> mouseListenerStack = new ArrayList<>();
   protected List<Board> boards = new CopyOnWriteArrayList<>();
   protected int[][] boardWidths; // Cache of board widths by row/column
   protected int[][] boardHeights; // Cache of board heights by row/column
   protected PieceCollection pieces = new DefaultPieceCollection();
   protected Highlighter highlighter = new ColoredBorder();
-  protected ArrayList<Highlighter> highlighters = new ArrayList<>();
+  protected List<Highlighter> highlighters = new ArrayList<>();
```

Comment:
These are going to cause a binary incompatibility I think, and we know Map is subclassed by some modules.

Reply:
oops - I get sucked in by the IDE telling me "chaaaaange this! chaaaaaange this!"